internal/config/config.go

```diff
@@ -35,6 +36,7 @@ type Configuration struct {
 	A3m           a3m.Config
 	AM            am.Config
 	API           api.Config
+	Archive       archive.Config
```

Comment:
Can we add a configuration example (even if it is with the defaults) to `enduro.toml`? I think a section called `[archive]` may not be clear enough, so we may want to change this field to `ExtractArchiveActivity` or something like that.